respy/pre_processing/process_covariates.py

```diff
@@ -83,12 +85,7 @@ def separate_covariates_into_core_dense_mixed(options, optim_paras):
         + [f"exp_{choice}" for choice in optim_paras["choices_w_exp"]]
         + [f"lagged_choice_{i}" for i in range(1, optim_paras["n_lagged_choices"] + 1)]
     )
-    dense_covs = set(optim_paras["observables"])
```

Comment:
So Tobi removes this step and instead uses the `create_dense_state_space_columns`. But there we also have that the dense columns are observables and types. Is this just a simplification?!
Shouldn't we add here the exogenous processes? @mo2561057